apps/ledger-live-mobile/src/newArch/features/WalletSync/screens/DeviceSelection/index.tsx

```diff
@@ -71,7 +65,7 @@ const WalletSyncActivationDeviceSelection: React.FC<ChooseDeviceProps> = ({
     // and avoids a duplicated error drawers/messages.
     // The only drawback: the user has to select again their device once the bluetooth requirements are respected.
     if (error instanceof BluetoothRequired) {
-      setDevice(undefined);
+      selectDevice(undefined);
```

Comment:
why not null instead of undefined ?

Reply:
I'm not sure TBH... I don't fully understand the comment above:
```
// Both the old (SelectDevice) and new (SelectDevice2) device selection components handle the bluetooth requirements with a hook + bottom drawer.
// By setting back the device to `undefined` it gives back the responsibilities to those select components to check for the bluetooth requirements
// and avoids a duplicated error drawers/messages.
// The only drawback: the user has to select again their device once the bluetooth requirements are respected.
```
I didn't take the time to dive into the `SelectDevice2` component logic but since the comment seems to insist on `undefined` I kept it.

Reply:
okay